broker-node/app/ChunkEvents.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+
+namespace App;
+
+
+
+use App\Clients\BrokerNode;
+
+use Illuminate\Database\Eloquent\Model;
+
+use Webpatser\Uuid\Uuid;
+
+
+
+class ChunkEvents extends Model
+
+{
+
+
+    public static function boot()
+    {
+
+        parent::boot();
+
+        self::creating(function ($model) {
+
+            $model->id = (string)Uuid::generate(4);
+
+        });
+
+    }
+    
+    protected $table = 'chunk_events';
+    protected $fillable = [
+        'hook_node_id',
+        'session_id',
+        'event_name',
+        'value'
+    ];
+    public $incrementing = false;  // UUID
+    
+
+	
+	public static function addChunkEvent($event_name, $hook_node_id, $session_id, $value){
+	
+		self::create([
+
+		    'hook_node_id' => $hook_node_id,
```

Comment:
This is super nit-picky, but it might be better to call this 'hooknode_id' instead of 'hook_node_id'.  hooknode_id is what we are using elsewhere (in the DataMap and CheckChunkStatus files).  Alternatively, we should change it to hook_node_id in those other places.